src/main/java/com/google/codeu/servlets/UserFormServlet.java

```diff
@@ -81,15 +81,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
         for (String tea : teaNames) {
           try {
             String formTea = request.getParameter(tea);
-            if (formTea == null || !formTea.isEmpty()) {
-              throw new NumberFormatException();
+            if (Strings.isNullOrEmpty(formTea)) {
```

Comment:
I am parsing through the parameters and checking if they are null or empty using Guava's null or empty function and throwing a exception if it's true. I am getting an ```500`` error that is caused by a NullPointerException 
Datastore.java:192
UserFormServlet.java:100

Am I going to have to put a stop on this for now and have the form validation done client side?

Reply:
Rather than throw an exception, just set the response status and return from `doGet`.